### PR TITLE
add emr in project

### PR DESCRIPTION
**Pull request by clhlc**
Hi，i add emr in plumbun and some sample_templates，i already test it and collectd emr data from cloudwatch to graphite.

### CODE REVIEW

**Comment by crccheck** [comment]
+1 I visually inspected the diff, but did not try it locally.

File: sample_templates/emr.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,73 @@
+Auth:
+  region: "{{ region }}"
+Metrics:
+{% for emr in resources %}
+{% for metric_name in (
+    'AppsCompleted',
```
Comment (paul-krohn) [inline sample_templates/emr.yml.j2]: You can put this list under `MetricName`, and drop the `for` loop. It won't make any functional difference (i.e. fewer API calls), but your config will be 1/28th as verbose.

File: sample_templates/emr.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  MetricName: "{{ metric_name }}"
+  Statistics:
+  - "Average"
+  Unit: "Count"
+  Dimensions:
+    JobFlowId: {{ emr.id }}
+{% endfor %}
+
+{% for metric_name in (
+    'MemoryAllocatedMB',
```
Comment (paul-krohn) [inline sample_templates/emr.yml.j2]: Same here, and below for `LiveDataNodes` and `HDFSUtilization`.

File: sample_templates/emr.yml.j2 [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    'PendingDeletionBlocks',
+    'S3BytesRead',
+    'S3BytesWritten',
+    'TotalLoad',
+    'UnderReplicatedBlocks',) %}
+- Namespace: "AWS/ElasticMapReduce"
+  MetricName: "{{ metric_name }}"
+  Statistics:
+  - "Average"
+  Unit: "Count"
```
Comment (paul-krohn) [inline sample_templates/emr.yml.j2]: It isn't necessary to name then units, unless there are statistics that, for some reason, you don't want returned.

**Comment by paul-krohn** [comment]
👍 on the code.

It would also be nice to have some example usage in the README about limitations and workarounds, such as for periodic jobs the have a different ID every run.

File: sample_templates/emr.yml.j2 (hunk 1 above)
Comment (clhlc) [inline sample_templates/emr.yml.j2]: thx,i will fix emr sample like kinisis, put matric name in the set?

**Pull request merged**